index.src.html

```diff
@@ -655,6 +655,27 @@ <h2 id="integration-with-html">Integration with HTML</h2>
   with W3C HTML5, those would need to be copied from [[HTML]].
 </section>
 
+<section class="informative">
+  <h2 id="integration-with-css">Integration with CSS</h2>
+
+  <em>This section is not normative.</em>
+
+  The CSS Standard does not specify how it fetches resources referenced from
+  stylesheets. However, implementations use the URL a given stylesheet was
+  loaded from as referrer for the requests for resources form that stylesheet.
+
+  Implementations should keep track of a referrer policy for each stylesheet
+  that should be used to create requests for resources from the respective
+  stylesheet. The referrer policy should be inherited from the entity that
+  triggered the load of the stylesheet, i.e., a document, or another
```

Comment:
@annevk pointed out that gecko uses the stylesheet's URL as referrer. I'd argue that the main point should be that the policy comes from the same place as the referrer. So if the stylesheet is the referrer, then the only way to get the referrer policy was via the referrer policy header.